src/tasks/harbor/harbor.ts

```diff
@@ -84,39 +89,46 @@ const systemRobot: any = {
 
 const robotPrefix = 'otomi-'
 const config: any = {
+  // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
can we just put this to the top of the file? repeating the same exclusion so many times is unnecessary

Reply:
sure, then we will ignore camelcase  in the whole file